### PR TITLE
[New Rule] Authorization Plugin Modification

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/855

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/credential_access_persistence_authorization_plugin_creation.toml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+]
+risk_score = 47
+rule_id = "e6c98d38-633d-4b3e-9387-42112cd5ac10"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Persistence", "Credential Access"]
+type = "query"
+
+query = '''
+event.category:file and not event.type:deletion and
+  file.path:(/Library/Security/SecurityAgentPlugins/* and not /Library/Security/SecurityAgentPlugins/TeamViewerAuthPlugin.bundle/Contents/*)
```
Comment (bm11100) [inline rules/macos/credential_access_persistence_authorization_plugin_creation.toml:30]:
```suggestion
event.category:file and not event.type:deletion and
  file.path:(/Library/Security/SecurityAgentPlugins/* and
  not /Library/Security/SecurityAgentPlugins/TeamViewerAuthPlugin.bundle/Contents/*)
```

Comment (bm11100) [reply rules/macos/credential_access_persistence_authorization_plugin_creation.toml:30]: just splitting the query after `and`

File: rules/macos/credential_access_persistence_authorization_plugin_creation.toml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+creation_date = "2021/01/13"
+maturity = "production"
+updated_date = "2021/01/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Authorization plugins are used to extend the authorization services API and implement mechanisms that are not natively
+supported by the OS such as multi-factor authentication with 3rd party software. Adversaries may abuse this feature to
+persist and/or collect clear text credentials as they traverse the registered rogue plugin during user logon.
```
Comment (bm11100) [inline rules/macos/credential_access_persistence_authorization_plugin_creation.toml:11]:
```suggestion
Authorization plugins are used to extend the authorization services API and implement mechanisms that are not natively
supported by the OS, such as multi-factor authentication with third party software. Adversaries may abuse this feature to
persist and/or collect clear text credentials as they traverse the registered plugins during user logon.
```

**Review by bm11100 — APPROVED** [review]

**Review by brokensound77 — APPROVED** [review]
LGTM

**Pull request merged**